src/plugins/orangehrmI18NPlugin/config/routes.yaml

```diff
@@ -2,3 +2,10 @@ core_i18n_messages:
   path: /core/i18n/messages
   controller: OrangeHRM\I18N\Controller\I18NMessagesController::handle
   methods: [ GET ]
+
+apiv2_i18_i18languages:
+  path: /api/v2/i18/i18-languages
+  controller: OrangeHRM\Core\Controller\Rest\V2\GenericRestController::handle
+  methods: [ GET ]
+  defaults:
+    _api: OrangeHRM\I18N\Api\I18NLanguageAPI
```

Comment:
add a new line at the end of the file, for all files which have below sign
![image](https://user-images.githubusercontent.com/25487187/172754323-d7266ddb-6a0c-42bf-ac62-c19359406497.png)